app/controllers/org_admin/templates_controller.rb

```diff
@@ -386,20 +386,15 @@ def template_options()
               end
             end
           end
+        else
+          # If no funder was selected retrieve the Org's templates
+          templates << Template.organisationally_visible.valid.where(published: true, org_id: org_id, customization_of: nil).to_a
```

Comment:
if org_id is blank, this else is executed too. Is there any problem for that query scope?